tests/components/cync/conftest.py

```diff
@@ -0,0 +1,31 @@
+"""Common fixtures for the Cync tests."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, create_autospec, patch
+
+import pycync
+import pytest
+
+from .const import MOCKED_EMAIL, MOCKED_USER
+
+
+@pytest.fixture(autouse=True)
+def client():
+    """Mock a pycync.Auth client."""
+    client_mock = create_autospec(pycync.Auth, instance=True)
+    client_mock.user = MOCKED_USER
+    client_mock.username = MOCKED_EMAIL
+    client_mock.login = AsyncMock()
+
+    with patch("homeassistant.components.cync.config_flow.Auth") as sc_class_mock:
+        sc_class_mock.return_value = client_mock
+        yield client_mock
```

Comment:
```suggestion
async def client() -> AsyncGenerator[AsyncMock]:
    """Mock a pycync.Auth client."""
    with patch("homeassistant.components.cync.config_flow.Auth", autospec=True) as sc_class_mock:
        client_mock = sc_class_mock.return_value
        client_mock.user = MOCKED_USER
        client_mock.username = MOCKED_EMAIL
        yield client_mock
```
It would also be great if we can add some kind of tests for the light platform and just that we create the right entities with the right data

Reply:
Added a test that checks the attributes of a few light devices on setup